src/dev/mergince.md

```diff
@@ -47,58 +47,64 @@ Settings about your deployment. Variables marked with star ⭐️ need to be mod
 | `COLLECT_STATISTICS`     | Boolean   | `true`    | Whether to send usage statistics for application improvements. |
 | `SERVICE_ID`             | string    |           | Deployment UUID. Auto-generated on first run. |
 ​
-
-#### Security settings
-Security settings. Variables marked with star ⭐️ need to be modified.
+#### Security settings 🛡️
+Security settings are important for production use.
 
 | Variable name            | Type      | Default   | Description |
```

Comment:
maybe we should sort them out alphabetically

Reply:
Do you mean environment variables or these sections?

Reply:
env variables for better readability